client/lib/mixins/analytics/index.js

```diff
@@ -69,7 +69,7 @@ const EVENTS = {
 			analytics.tracks.recordEvent( 'calypso_domain_search_results_mapping_button_click', { section } );
 		},
 
-		searchFormSubmit( searchBoxValue, section, timeDiffFromLastSearch, searchCount, searchVendor ) {
```

Comment:
I'm not GA expert, but maybe it'd be best to leave the `search_vendor` there too? It will be constant, but it will also mean the data will stay consistent - otherwise if someone were to analyze this data, they will have to remember that an empty `search_vendor` during the X period of time means it's domainsbot, etc.